code/modules/recycling/package_wrapper.dm

```diff
@@ -186,7 +188,7 @@
 	desc        = "A tube... of cardboard."
 	icon        = 'icons/obj/items/gift_wrapper.dmi'
 	icon_state  = "c_tube"
-	w_class     = ITEM_SIZE_NORMAL
+	w_class     = ITEM_SIZE_TINY
```

Comment:
Wrapping paper tubes are pretty damn big, like anywhere from 1ft to 4ft (0.3m to 1.2m) long. It should probably have the same w_class as the package/gift wrap.